src/components/ContentOpenWith/ContentOpenWith.js

```diff
@@ -389,15 +396,32 @@ class ContentOpenWith extends PureComponent<Props, State> {
                             integrations={integrations}
                         />
                     )}
-                    {shouldRenderIntegrationPortal && (
+                    {(shouldRenderLoadingIntegrationPortal ||
+                        shouldRenderErrorIntegrationPortal) && (
                         <IntegrationPortal
                             integrationWindow={this.integrationWindow}
+                            key={uniqueid('integration_portal_')}
```

Comment:
Will this create a new unique ID per-render? Is that desired or should it be tied to the integration somehow?

Reply:
You can open the same integration multiple times in the error case. So that needs to cause a re-render

Reply:
As per our discussion, switching to a regular `Component`